plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/settings/CodeWhispererConfigurable.kt

```diff
@@ -87,6 +92,20 @@ class CodeWhispererConfigurable(private val project: Project) :
                     bindSelected(codeWhispererSettings::isImportAdderEnabled, codeWhispererSettings::toggleImportAdder)
                 }.comment(message("aws.settings.codewhisperer.automatic_import_adder.tooltip"))
             }
+
+            row {
+                link("Configure inline suggestion keybindings") { e ->
+                    val settings = DataManager.getInstance().getDataContext(e.source as ActionLink).getData(Settings.KEY) ?: return@link
+                    val configurable: Configurable = settings.find("preferences.keymap") ?: return@link
```

Comment:
user needs feedback if these are null

Reply:
will add a TODO for now, can't think of a best way to inform users failures in this case